src/emulator/downloadableEmulators.ts

```diff
@@ -323,7 +323,12 @@ export function _getCommand(
   args: { [s: string]: any },
 ): DownloadableEmulatorCommand {
   const baseCmd = Commands[emulator];
-
+  // If a commandOverride is passed, we use that instead but pass in the optional args.
+  const commandOverride = process.env[`${emulator.toUpperCase()}_EMULATOR_COMMAND`];
+  if (commandOverride) {
+    baseCmd.binary = commandOverride;
+    baseCmd.args = [];
```

Comment:
@yuchenshi could you sanity check me on this? I'm pretty sure we need to omit these required args for the 'blaze run' case to work - take a look at the bae commands for the java emulators, and see how we pass in java args that won't work with 'blaze run' IIUC.